poweriq-worker/src/main/java/com/vmware/flowgate/poweriqworker/jobs/PowerIQService.java

```diff
@@ -119,17 +118,17 @@
       subCategoryMap.put(SmokeSensor, AssetSubCategory.Smoke);
       subCategoryMap.put(WaterSensor, AssetSubCategory.Water);
       subCategoryMap.put(Vibration, AssetSubCategory.Vibration);
-      sensorType.add(ServerSensorType.HUMIDITY);
-      sensorType.add(ServerSensorType.BACKPANELTEMP);
-      sensorType.add(ServerSensorType.FRONTPANELTEMP);
-      sensorValueType.put(HumiditySensor, MetricName.HUMIDITY);
-      sensorValueType.put(TemperatureSensor, MetricName.TEMPERATURE);
-      sensorValueType.put(AirFlowSensor, MetricName.AIR_FLOW);
-      sensorValueType.put(AirPressureSensor, MetricName.AIRPRESSURE);
-      sensorValueType.put(ContactClosureSensor, MetricName.CONTACTCLOSURE);
-      sensorValueType.put(SmokeSensor, MetricName.SMOKE);
-      sensorValueType.put(WaterSensor, MetricName.WATER_FLOW);
-      sensorValueType.put(Vibration, MetricName.VIBRATION);
+      metricsName.add(MetricName.SERVER_HUMIDITY);
+      metricsName.add(MetricName.SERVER_BACK_TEMPREATURE);
```

Comment:
You can define a group for temperature/Humidty metric name for PDU even with the same value. 

Like PDU_INLET_TEMERATURE  PDU_OUTLET_TEMPERATURE  PDU_EXTERNAL_TEMPERATURE

same as Humidity.